src/services/state-service.ts

```diff
@@ -14,33 +14,28 @@ export class GlobalExtensionState {
 }
 export type GlobalExtensionStateKey = keyof GlobalExtensionState;
 
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
 export class LocalExtensionState {
-  public AuthenticatingInProgress = false;
-  public HasAnyDetections = false;
-  public TreeViewIsOpen = false;
+  // add local state here
```

Comment:
Are these comments things that we want to implement later or just for documentation purposes? Maybe I'm missing context
 
```suggestion
  // TODO: add local state here
```

Reply:
it was implemented during refactoring phase but after doing more refactoring stuff it is not used at all right now; so it was for migration purposes; but i dont want to remove functionality to work with local state because one day we could want to use it again; so not TODO, but general comment :)

Reply:
Ah, I see. Thanks for the context; makes sense :)